src/components/Footer.js

```diff
@@ -20,9 +20,9 @@ export const FooterTemplate = ({
         <ul className="footer-column-1">
           <li className="keep-logo">Keep</li>
           <li>
-            <Button className="email-btn" href={`mailto:${emailLink.email}`}>
+            <button className="email-btn" href={`mailto:${emailLink.email}`}>
```

Comment:
If we're `href`ing we should really use an `a` :/

Reply:
Ah, yeah---it actually looks like this doesn't work with a `button`, at least in Safari.

Reply:
Looks like this button also shrank in size---may have been on purpose, just flagging it.

Reply:
Whoops! Sorry I was stuck in button land! 😆  Updated.